### ISSUE TITLE
Skulls with texture set with setOwnerProfile and Bukkit.createProfile will not be shown

### ISSUE THREAD

**Issue by PierreSchwang** [DecentSoftware-eu/DecentHolograms#177]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

When calling `DHAPI.insertHologramLine(Hologram, int, ItemStack)` with a player skull, it'll not work if the skulls texture is set by using Papers / Bukkits [SkullMeta#setOwnerProfile](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/inventory/meta/SkullMeta.html#setOwnerProfile(org.bukkit.profile.PlayerProfile))

The problem is in the HologramItem class: https://github.com/DecentSoftware-eu/DecentHolograms/blob/673e45c724910021d47f23257845c659cb60db17/src/main/java/eu/decentsoftware/holograms/api/utils/items/HologramItem.java#L137-L143

It should be checked first, if there is an actual texture set and then fallback to the profile name. When setting the profile using a simple profile created:
```java
this.profile = Bukkit.createProfile(STATIC_UUID);
try {
    PlayerTextures textures = this.profile.getTextures();
    textures.setSkin(new URL(String.format(URL_TEMPLATE, value)));
    this.profile.setTextures(textures);
} catch (MalformedURLException e) {
    throw new IllegalArgumentException("Invalid skull data for " + value, e);
}
// ...
ItemStack itemStack = new ItemStack(Material.PLAYER_HEAD);
SkullMeta meta = (SkullMeta) itemStack.getItemMeta();
if (profile != null) {
    meta.setOwnerProfile(this.profile);
}
itemStack.setItemMeta(meta);
```

Internally, `CraftServer#createProfile` instantiates a new CraftPlayerProfile. The constructor populates its actual GameProfile with an empty-string (`""`) name if the name passed is null (as done by the original method). The name will always be non-null, even when not passing one or using the texture. 

### Solution

Switch the if-else-if conditions and corresponding calls in https://github.com/DecentSoftware-eu/DecentHolograms/blob/673e45c724910021d47f23257845c659cb60db17/src/main/java/eu/decentsoftware/holograms/api/utils/items/HologramItem.java#L137-L143
so that the texture itself has a higher priority than the name of the GameProfile

### Server Version

Paper 1.20.2

### Client Version

1.20.2

### Plugin Version

2.8.5

### Log

-

**Comment by d0by1** [comment]
#169 seems to fix this. Thanks for reporting this and I will make sure to include the fix in the next release.

**Comment by PierreSchwang** [comment]
> #169 seems to fix this. Thanks for reporting this and I will make sure to include the fix in the next release.

Oh sorry, I have not seen that PR while searching through the Repo. Thanks for the information!

**Comment by d0by1** [comment]
Fixed in 2.8.6.
